packages/create-emotion/types/index.d.ts

```diff
@@ -1,107 +1,122 @@
 }
 
 export type Interpolation =
-  | undefined | null | boolean | string | number
+  | undefined
+  | null
+  | boolean
+  | string
+  | number
   | TemplateStringsArray
   | CSSObject
   | ArrayInterpolation
   | ClassInterpolation
-  ;
 
 export interface ArrayClassNameArg extends Array<ClassNameArg> {}
 
 export type ClassNameArg =
-  | undefined | null | boolean | string
+  | undefined
+  | null
+  | boolean
+  | string
   | { [key: string]: undefined | null | boolean | string }
   | ArrayClassNameArg
-  ;
 
 export interface StyleSheet {
-  inject(): void;
-  speedy(bool: boolean): void;
-  insert(rule: string, sourceMap?: string): void;
-  flush(): void;
+  inject(): void
+  speedy(bool: boolean): void
+  insert(rule: string, sourceMap?: string): void
+  flush(): void
 }
 
 export interface EmotionCache {
   registered: {
-    [key: string]: string;
-  };
+    [key: string]: string
+  }
   inserted: {
-    [key: string]: string;
-  };
-  nonce?: string;
-  key: string;
+    [key: string]: string
+  }
+  nonce?: string
+  key: string
 }
 
 export interface Emotion {
-  flush(): void;
-  hydrate(ids: Array<string>): void;
-  cx(...classNames: Array<ClassNameArg>): string;
-  merge(className: string, sourceMap?: string): string;
-  getRegisteredStyles(registeredStyles: Array<string>, classNames: string): string;
-  css(...args: Array<Interpolation>): string;
-  injectGlobal(...args: Array<Interpolation>): void;
-  keyframes(...args: Array<Interpolation>): string;
-  sheet: StyleSheet;
-  caches: EmotionCache;
+  flush(): void
+  hydrate(ids: Array<string>): void
+  cx(...classNames: Array<ClassNameArg>): string
+  merge(className: string, sourceMap?: string): string
+  getRegisteredStyles(
+    registeredStyles: Array<string>,
+    classNames: string
+  ): string
+  css(...args: Array<Interpolation>): string
+  injectGlobal(...args: Array<Interpolation>): void
+  keyframes(...args: Array<Interpolation>): string
+  sheet: StyleSheet
+  caches: EmotionCache
 }
 
 export interface EmotionBaseContext {
-  __SECRET_EMOTION__?: Emotion;
+  __SECRET_EMOTION__?: Emotion
 }
 
 export interface EmotionContext extends EmotionBaseContext {
-  [key: string]: any;
+  [key: string]: any
 }
 
 export type StylisPlugins =
   | null
   | ((...args: Array<any>) => any)
   | Array<(...args: Array<any>) => any>
-  ;
 
 export interface EmotionOptions {
-  nonce?: string;
-  stylisPlugins?: StylisPlugins;
-  prefix?: boolean | ((key: string, value: string, context: 1 | 2 | 3) => boolean);
-  key?: string;
-  container?: HTMLElement;
+  nonce?: string
+  stylisPlugins?: StylisPlugins
+  prefix?:
+    | boolean
+    | ((key: string, value: string, context: 1 | 2 | 3) => boolean)
+  key?: string
+  container?: HTMLElement
 }
 
-export default function createEmotion(context: EmotionContext, options?: EmotionOptions): Emotion;
+export default function createEmotion(
+  context: EmotionContext,
+  options?: EmotionOptions
+): Emotion
```

Comment:
The `context` argument here was removed btw.

Reply:
Actually, there are more broken types for those pre-typed packages, but I think fixing those could be handled separately.